endpoints/events/event.go

```diff
@@ -26,10 +26,11 @@ const (
 	AccountIdParameter = "a"
 
 	// Optional
-	BidderParameter    = "bidder"
-	TimestampParameter = "ts"
-	FormatParameter    = "f"
-	AnalyticsParameter = "x"
+	BidderParameter          = "bidder"
+	TimestampParameter       = "ts"
+	FormatParameter          = "f"
+	AnalyticsParameter       = "x"
+	IntegrationTypeParameter = "int"
```

Comment:
Just checking, is this a standardized query parameter agreed upon with the PBS committee?

Reply:
Yes I believe so, I'll double check with Mansi, but I believe this is correct

Reply:
I see that this is the name of the parameter in PBS-Java.